### PR TITLE
Reordered Changelog for readability, fixed typos

### PR DESCRIPTION
**Pull request by MatGB**
~ Split the general / GUI list up into several sublists and organised entries by window
~ Fixed some typos

### CODE REVIEW

**Comment by MatGB** [comment]
I've reformatted some of the lists to follow Markdown properly and reorganised the very long GUI entry into several sublists to enhance readability, also squashed a few typos.

I've put this as a PR to the Release branch, on its own it's not worth doing another RC, but if we do need one then this should go in.

File: ChangeLog.md [hunk 1]
```diff
@@ -40,66 +40,85 @@ Notable changes to the FreeOrion project will be documented in this file.
...
+
+- Sidepanel
+    - Added automatic selection of ships for bombarding.
+    - Added ability to rename systems in SidePanel. System sorting considers the system's id to disambiguate if necessary.
+    - Displayed source planet in growth special accounting labels.
+    - Added indication of defending troops to invade button.
+    - In sidepanel, displayed tooltip not only for the selected focus, but also for available focus settings in the focus dropdown list.
+
+- Fleet Window
+    - Added more and more flexible indicator icons to fleet and ship panels.
```
Comment (LGM-Doyle) [inline ChangeLog.md]: suggestion ?
more and more -> greater variety and more

File: ChangeLog.md [hunk 2]
```diff
@@ -40,66 +40,85 @@ Notable changes to the FreeOrion project will be documented in this file.
...
+    - Labeled fleets window as "Near System X" for fleets in transit.
+    - Showed empire name in fleets window for fleets in transit.
+    - Added right-click menu command to split from a fleet ships that have less than full fighter complement.
+
+- SitRep Window
+    - Allowed SitRep right-click menu from majority of sitrep area.
+    - Added Copy to sitrep right-click context menu.
+    - Added link to help article on sitrep entry right click menu.
+    - Added right click context to hide/show types of SitReps.
+    - Changed combat log to use empire color to designate destroyed objects.
```
Comment (LGM-Doyle) [inline ChangeLog.md:102]: Is this line for the SitRep window or the combat log?

**Comment by dbenage-cx** [comment]
These are not your changes, noting them as marked `and typos` and not to have conflicting PR:

`Bugs`
Line 151/170:
`- Fixed window installer continuing install before uninstall completes.`
Windows

Line 180/199:
`- Fixed unonwed mines not destroying ships after reducing their structure to 0.`
Unowned

Line 192/212 (`Technical/Internal`) should remove or escape the formatting tags:
`- Implemented <sup> and <sub> text formatting tags.`
When rendered in markdown (e.g. on github) results in:
- Implemented <sup> and <sub> text formatting tags.

File: ChangeLog.md (hunk 2 above)
Comment (MatGB) [reply ChangeLog.md:102]: The latter, there was only one log change that I noticed and I meant to make this Sitrep / Combat log but forgot, as you've found other typos I missed while copy/pasting I'll do another pass in a bit.

**Comment by MatGB** [comment]
Fixed everything, thanks, especially for the eachother spots, I thought I'd got all of them.

**Comment by Vezzra** [comment]
Gave it only a cursory glance, looks good to me. As far as I'm concerned, LGTM.

**Comment by LGM-Doyle** [comment]
LGTM

**Pull request merged**